test/utils/utils.ts

```diff
@@ -6,7 +6,7 @@ import * as path from "path";
 let spawnSync: any = child.spawnSync;
 if (process.platform === 'win32') {
     spawnSync = (cmd: string, args: string[]) => {
-        return child.spawnSync('node', [cmd].concat(args));
+        return child.spawnSync(path.join(__dirname, '../../node-v8/bin/node.exe'), [cmd].concat(args));
```

Comment:
For construction like this path.join(__dirname, '../../<some path>') better using path.resolve